.github/workflows/test.yaml

```diff
@@ -0,0 +1,82 @@
+name: KBase Handle Service 2 test
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+  handle_service_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - python: '3.9.19'
+            mongo: 'mongodb-linux-x86_64-3.6.23'
+            wired_tiger: 'false'
+
+    steps:
+    - name: Check out GitHub repo
+      uses: actions/checkout@v4
+
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{matrix.python}}
+
+    - name: Install dependencies and set up test config
+      shell: bash
+      env:
+        AUTH_SERVICE_URL: https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login
+        AUTH_URL: https://ci.kbase.us/services/auth
+        SHOCK_URL: https://ci.kbase.us/services/shock-api
+        TEST_TOKEN: ${{secrets.KBASE_CI_TOKEN}}
+        ADMIN_TOKEN: ${{secrets.KBASE_CI_TOKEN}}
```

Comment:
diff token failed.

Reply:
I assume `KBASE_CI_TOKEN2` does not have admin right?

Reply:
Neither do. If the token needs admin rights then they both should fail

Reply:
It failed because of here:https://github.com/kbase/handle_service2/blob/aeb1a30ba9285fe4a15bfa19d7ec6d20336ebe98/lib/AbstractHandle/Utils/ShockUtil.py#L115-L116

use user2 token (`KBASE_CI_TOKEN2 `) to grand readable ACL of user1 node is not gonna work.

Reply:
Ok, you should check with @Tianhao-Gu to see how these tests passed before

Reply:
It passed because he used one token consistently

Reply:
What about the test token from test_local/test.cfg?

Reply:
I never use test_local/test.cfg for handle_service2

Reply:
If you ran kb-sdk test you must've

Reply:
Hi @Tianhao-Gu, could you please weight in here to clarify?

Reply:
I use one token in `test.cfg` file. `kb_sdk_test.yaml` lists step of how to config kb-sdk tests before.

Reply:
So testing the admin user setting another user's permissions isn't actually tested?

Reply:
Per discussion in slack, we use same token for admin and test. 
An issue is also created here: https://github.com/kbase/handle_service2/issues/112

Reply:
:+1: